files/build_templates/qos_config.j2

```diff
@@ -0,0 +1,209 @@
+{%- set PORT_ALL = [] %}
+{%- for port in PORT %}
+    {%- if PORT_ALL.append(port) %}{% endif %}
+{%- endfor %}
+{%- if PORT_ALL | sort_by_port_index %}{% endif %}
+
+{%- set port_names_list_all = [] %}
+{%- for port in PORT_ALL %}
+    {%- if port_names_list_all.append(port) %}{% endif %}
+{%- endfor %}
+{%- set port_names_all = port_names_list_all | join(',') -%}
+
+
+{%- set PORT_ACTIVE = [] %}
+{%- if DEVICE_NEIGHBOR is not defined %}
+    {%- set PORT_ACTIVE = PORT_ALL %}
+{%- else %}
+    {%- for port in DEVICE_NEIGHBOR.keys() %}
+        {%- if PORT_ACTIVE.append(port) %}{%- endif %}
+    {%- endfor %}
+{%- endif %}
+{%- if PORT_ACTIVE | sort_by_port_index %}{% endif %}
+
+{%- set port_names_list_active = [] %}
+{%- for port in PORT_ACTIVE %}
+    {%- if port_names_list_active.append(port) %}{%- endif %}
+{%- endfor %}
+{%- set port_names_active = port_names_list_active | join(',') -%}
+
+
+{
+    "TC_TO_PRIORITY_GROUP_MAP": {
```

Comment:
Hi @mkbalani, This is actually the question I have for you. I see TC_TO_PRIORITY_GROUP_MAP to have only 3 and 4 on barefoot. We cannot include other mappings: "0": "0", "1": "1", "2": "2", "5": "5", "6": "6", "7": "7" here? Can you elaborate the reasons?

    "TC_TO_PRIORITY_GROUP_MAP": {
        "AZURE": {
            "3": "3",
            "4": "4" 
        }   
    },

Reply:
I have customized TC_TO_PRIORITY_GROUP_MAP and PFC_PRIORITY_TO_PRIORITY_GROUP_MAP for barefoot devices. Can you check again?

Reply:
I do not understand why customization is needed for barefoot platform. @mkbalani , can you elaborate? @wendani , we should not do customization unless there is legitimate reason, otherwise it is too difficult the manage the complexity here.